### ISSUE TITLE
unFair comparison in MNIST tutorial

### ISSUE THREAD

**Issue by HuangChiEn** [tensorflow/quantum#629]
As the last sentence "One way or the other, the classical neural network easily outperforms the quantum neural network. For classical data, it is difficult to beat a classical neural network." leaved in the MNIST tutorial, I may have some wondering.

 
The classical model(with 37 params) has the 128 batch_size as well as the 20 epochs to achieve more than 90% accuracy. However, the quantum model only runs on 32 batch_size and 3 epochs. When I run the test by setting the 'same (32 batch_size, 3 epochs)' configuration on the classical model, the quantum model has achieved about 50~66% accuracy which outperforms the classical model (49%). 

Quantum Model :
![image](https://user-images.githubusercontent.com/52521165/134657895-3c494e00-2fad-482e-b425-c13a85ac38ad.png)
![image](https://user-images.githubusercontent.com/52521165/134657932-9c39117a-4e43-4792-bc1b-295d29220276.png)

Classical Model : 
![image](https://user-images.githubusercontent.com/52521165/134657747-99d2a536-bbbf-4847-bfea-527b8eecfe16.png) 
![image](https://user-images.githubusercontent.com/52521165/134657824-38393bce-b4af-48b8-8548-495ca56dc2f1.png)

---

Result : 
![image](https://user-images.githubusercontent.com/52521165/134657618-303a97f8-89f3-438d-b2c9-901aeb89e1b7.png)

Maybe we should reconsider the conclusion of MNIST tutorial (at least, both of them should have the same ability).

**Comment by lockwo** [comment]
I think the meaning of the sentence is that fully trained classical models (even trivial MLPs) will pretty much always outperform QML models. The only reason the quantum model is trained so quickly is just for the brevity of the tutorial, the authors note "Training this model to convergence should achieve >85% accuracy on the test set". Thus both models are trained to completion, classical models prevail. After 3 epochs neither models is done training, so it's not a very meaningful statement to say one is outperforming the other.

Additionally, with your example of running 3 epochs for the classical model, I imagine there is substantial variance in the performance. Because so few gradient steps are taken, my intuition tells me the random seed plays a significant role given the importance of initial proximity to decent local minima that is now important with limited updates. A quick run of my own seems to indicate that. I ran the same conditions and got >85% test accuracy with the classical model in just 3 epochs (better than the QML model would do with many epochs. 
![image](https://user-images.githubusercontent.com/42878312/134751925-57844dac-d3b0-4fd7-9c16-6fc701b2032b.png)

**Comment by HuangChiEn** [comment]
thanks for sharing your perspective~~ it's nice~
